beetmoverscript/docker.d/worker.yml

```diff
@@ -11,8 +11,6 @@ taskcluster_scope_prefixes:
         - 'project:releng:beetmover:'
       'COT_PRODUCT == "thunderbird"':
         - 'project:comm:thunderbird:releng:beetmover:'
-      'COT_PRODUCT == "mobile"':
-        - 'project:mobile:firefox-android:releng:beetmover:'
```

Comment:
We should also remove all the other mobile cot-product bits in this file and init_worker.sh.